aspnetcore/index.yml

```diff
@@ -0,0 +1,455 @@
+          itemType: overview
+          text: "Overview"
+        - url: blazor/get-started.md
+          itemType: get-started
+          text: "Create your first Blazor app"
+        - url: tutorials/build-your-first-blazor-app.md
+          itemType: get-started
+          text: "Build your first Blazor app with reusable components"
+        - url: blazor/hosting-models.md
+          itemType: concept
+          text: "Blazor hosting models"
+    # Card
+    - title: RESTful web API apps
+      summary: Develop RESTful HTTP services with ASP.NET Core web API
+      links:
+        - url: web-api/index.md
+          itemType: overview
+          text: "Overview"
+        - url: tutorials/first-web-api.md
+          itemType: get-started
+          text: "Create your first web API with ASP.NET Core"
+        - url: tutorials/web-api-help-pages-using-swagger.md
+          itemType: tutorial
+          text: "Generate web API help pages with Swagger / OpenAPI"
+        - url: web-api/action-return-types.md
+          itemType: concept
+          text: "Controller action return types"
+        - url: web-api/advanced/formatting.md
+          itemType: concept
+          text: "Format response data"
+        - url: web-api/handle-errors.md
+          itemType: concept
+          text: "Handle errors"
+        - url: tutorials/web-api-javascript.md
+          itemType: tutorial
+          text: "Call an ASP.NET Core web API with JavaScript"
+      # footerLink (optional)
+      footerLink:
+        url: web-api/index.md
+        text: "See more"
+    # Card
+    - title: Page-focused web UI with MVC
+      summary: "Develop web apps using the Model-View-Controller design pattern"
+      links:
+        - url: mvc/overview.md
+          itemType: overview
+          text: "Overview"
+        - url: tutorials/first-mvc-app/index.md
+          itemType: get-started
+          text: "Create your first ASP.NET Core MVC app"
+        - url: tutorials/first-mvc-app/adding-controller.md
+          itemType: tutorial
+          text: "Add a controller"
+        - url: tutorials/first-mvc-app/adding-view.md
+          itemType: tutorial
+          text: "Add a view"
+        - url: tutorials/first-mvc-app/adding-model.md
+          itemType: tutorial
+          text: "Add a model"
+        - url: tutorials/first-mvc-app/validation.md
+          itemType: tutorial
+          text: "Add validation"
+        - url: mvc/controllers/routing.md
+          itemType: concept
+          text: "Routing to controller actions"
+        - url: mvc/controllers/testing.md
+          itemType: concept
+          text: "Unit test"
+      # footerLink (optional)
+      footerLink:
+        url: mvc/overview.md
+        text: See more
+      # Card
+    - title: "Page-focused web UI with Razor Pages"
+      summary: "Develop page-focused web apps with a clean separation of concerns"
+      links:
+        - url: razor-pages/index.md
+          itemType: overview
```

Comment:
* RP should go before MVC as Scott mentioned.
`url: razor-pages/index.md` is not an overview. I'd move it after  `"Create your first Razor Pages web app"`. It's an advanced introduction, not an overview.